packages/dashmate/src/commands/setup.js

```diff
@@ -129,15 +129,15 @@ SetupCommand.args = [{
 }];
 
 SetupCommand.flags = {
-  'debug-logs': flagTypes.boolean({
+  'debug-logs': Flags.boolean({
     char: 'd', description: 'enable debug logs', allowNo: true, default: null,
   }),
-  'external-ip': flagTypes.string({ char: 'i', description: 'external ip' }),
-  'operator-bls-private-key': flagTypes.string({ char: 'k', description: 'operator bls private key' }),
-  'funding-private-key': flagTypes.string({ char: 'p', description: `private key with more than ${MASTERNODE_DASH_AMOUNT} dash for funding collateral` }),
-  'node-count': flagTypes.integer({ description: 'number of nodes to setup', default: null }),
-  'miner-interval': flagTypes.string({ char: 'm', description: 'interval between blocks', default: null }),
-  verbose: flagTypes.boolean({ char: 'v', description: 'use verbose mode for output', default: false }),
+  'external-ip': Flags.string({ char: 'i', description: 'external ip' }),
+  'operator-bls-private-key': Flags.string({ char: 'k', description: 'operator bls private key' }),
+  'funding-private-key': Flags.string({ char: 'p', description: `private key with more than ${MASTERNODE_DASH_AMOUNT} dash for funding collateral` }),
+  'node-count': Flags.integer({ description: 'number of nodes to setup', default: undefined }),
+  'miner-interval': Flags.string({ char: 'm', description: 'interval between blocks', default: undefined }),
```

Comment:
@shumkov this changing of the default value from `null` to `undefined` was the bug that had me stuck for the longest time. The new oclif integer parser will attempt to [assign the default if it is not undefined](https://github.com/oclif/core/blob/main/src/parser/parse.ts#L208), which results in the flag type checker [failing because null is not an integer](https://github.com/oclif/core/blob/main/src/parser/flags.ts#L40). Can you see any problem with changing this, or should we just delete the default value? Should it be changed elsewhere?

Reply:
I would just remove the default values. Were there any checks for `null` in commands or related logic?

Reply:
Not for these commands. We often check for `null` when checking for the group or config name, but not for these flags. I've removed it.